internal/service/eks/addon_data_source.go

```diff
@@ -35,6 +35,11 @@ func DataSourceAddon() *schema.Resource {
 				Required:     true,
 				ValidateFunc: validClusterName,
 			},
+			"configuration_values": {
```

Comment:
We typically prefer to have separate pull requests for a resource and datasource.